### PR TITLE
Added self-service css support

### PR DESCRIPTION
**Pull request by PeculiarGoat**
Looks like this works okay but I'm going to hold off on review/merge until API issues are sorted.

### CODE REVIEW

**Comment by PeculiarGoat** [comment]
API seems to be working fine in staging now so this can be reviewed/merged.

**Review by brynyShift72 — COMMENTED** [review]
LGTM

**Review by brynyShift72 — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change log
 
+## Unreleased
+Added CSS api call.
```
Comment (stajs) [inline changelog.md:4]:
```suggestion
Added CSS API call.
```

**Review by stajs — APPROVED** [review]
LGTM

**Pull request merged**